### PR TITLE
only add default one time

### PR DESCRIPTION
**Pull request by sb-cecilialiu**
## PR Template

Jira Ticket: [MAT-5095](https://jira.cms.gov/browse/MAT-5095)
(Optional) Related Tickets:

### All Submissions
* [x] This PR has the JIRA linked.
* [ ] Required tests are included
* [x] No extemporaneous files are included (i.e. Complied files or testing results)
* [x] This PR is into the **correct branch**.
* [ ] All Documentation as needed for this PR is Complete (or noted in a TODO or other Ticket)
* [ ] Any breaking changes or failing automation are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [x] This PR has NO significant security impact (i.e. Changing auth methods, Adding a new user type, Adding a required but vulnerable package)

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
*  The tests appropriately test the new code, including edge cases
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads

### CODE REVIEW

File: lambda-function/src/utils/measureConversionUtils.ts [hunk 1]
```diff
@@ -214,13 +214,20 @@ const getMeasuretypes = (measuretypes: Array<MeasureType>): Array<MeasureGroupTy
         types.push(MeasureGroupTypes.STRUCTURE);
         break;
       default:
-        types.push(MeasureGroupTypes.OUTCOME);
+        if (!alreadyHasDefaultMeasureTye(types)) {
```
Comment (adongare) [inline lambda-function/src/utils/measureConversionUtils.ts]: because eventually it is just an array of strings, probably `array.includes` or `array.indexOf` would work

Comment (sb-cecilialiu) [reply lambda-function/src/utils/measureConversionUtils.ts]: Ok thanks changed to use includes.

**Review by adongare — APPROVED** [review]
